test/integration/database.spec.ts

```diff
@@ -161,9 +161,10 @@ describe('admin.database', () => {
 });
 
 function addValueEventListener(
-    // Check for type compilation
-    db: admin.database.Database,
-    callback: (s: admin.database.DataSnapshot) => any) {
-    let eventType: admin.database.EventType = 'value';
-    db.ref().on(eventType, callback);
+  // Check for type compilation. This method is not invoked by any tests. But it will
+  // trigger a TS compilation failure if the RTDB typings were not loaded correctly.
+  db: admin.database.Database,
+  callback: (s: admin.database.DataSnapshot) => any) {
```

Comment:
Didn't clarify. In terms of indents, it should be:
function name(
    arg1,
    arg2) {
  statements
}

4 space indents after (
and the statements within the function 2 space indented.

Reply:
Done